src/display/svg.js

```diff
@@ -982,8 +982,54 @@ SVGGraphics = class SVGGraphics {
    */
   _makeColorN_Pattern(args) {
     if (args[0] === 'TilingPattern') {
-      warn('Unimplemented pattern TilingPattern');
-      return null;
+      const color = args[1];
```

Comment:
Perhaps it's better to split the logic off to a new method named `_makeTilingPattern`, just like `_makeShadingPattern` below, for consistency.